tempesta_fw/ss_skb.c

```diff
@@ -734,7 +744,8 @@ ss_skb_get_room(struct sk_buff *skb, char *pspt, unsigned int len, TfwStr *it)
 int
 ss_skb_cutoff_data(const TfwStr *hdr, int skip, int tail)
 {
-	int r;
+	int r, f_len;
+	void *f_ptr;
```

Comment:
There is no sense to introduce the variable. As many variables you have to keep in mind, as harder the code is understand to.